src/nvim/screen.c

```diff
@@ -2081,6 +2058,151 @@ fill_foldcolumn (
   }
 }
 
+
+static int get_column_diff_until_eol(win_T* wp, int col) {
```

Comment:
I don't have a clear opinion here, I'll leave it as is unless someone strongly opposes to that name.